src/language-markdown/parser-markdown.js

```diff
@@ -96,6 +97,34 @@ function liquid() {
   };
 }
 
+function wikiLink() {
+  const entityType = "wikiLink";
+  const wikiLinkRegex = /^\[\[(?<linkContents>.+?)]]/s;
```

Comment:
Hope we don't break content using regexp here

Reply:
Can you clarify more what you're worried about?